datasets/HakaiQuadraLimpet5min.xml

```diff
@@ -221,20 +250,30 @@
       <att name="long_name">DepthQuadraLimpet_Avg</att>
       <att name="units">m</att>
       <att name="cell_methods">time: mean (interval: 5.0 minutes)</att>
-      <att name="ancillary_variables">WaterDepth_flags</att>
+      <att name="ancillary_variables">WaterDepth_flags,WaterDepth_UQL</att>
```

Comment:
It looks like coastwatch is using a space-separated list instead of comma for `ancillary_variables`, eg:
https://coastwatch.pfeg.noaa.gov/erddap/griddap/pmlEsaCCI31OceanColorDaily.html

Reply:
oh yeah you're right!